test/DurableTask.SqlServer.Tests/Utils/SharedTestHelpers.cs

```diff
@@ -29,7 +29,7 @@ public static string GetTestName(ITestOutputHelper output)
         public static string GetDefaultConnectionString(string database = "DurableDB")
         {
             // The default for local development on a Windows OS
-            string defaultConnectionString = $"Server=localhost;Database={database};Trusted_Connection=True;";
+            string defaultConnectionString = $"Server=localhost;Database={database};Trusted_Connection=True;Encrypt=False;";
```

Comment:
Can you clarify why we need to disable encryption?

Reply:
This should be the default behavior already for the version of the SQL SDK that we're using. Just making it explicit. In [the most recent version (4.0.0)](https://github.com/dotnet/SqlClient/blob/main/release-notes/4.0/4.0.0.md#breaking-changes), we'll be required to set this explicitly if we want our tests to work with arbitrary SQL container instances (like the one we use for the CI tests).